app/portainer/home/EnvironmentList/KubeconfigButton.css

```diff
@@ -0,0 +1,15 @@
+.center-dialog {
+  display: flex;
+  flex-direction: column;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
```

Comment:
no need for those, postcss will add the prefixes

Reply:
Removed